### PR TITLE
[WIP] Change `TerminatorKind::Assert` structure

### PR DESCRIPTION
**Pull request by saleemjaffer**

### CODE REVIEW

File: src/librustc/mir/mod.rs [hunk 1]
```diff
@@ -1150,7 +1150,7 @@ pub enum TerminatorKind<'tcx> {
     Assert {
         cond: Operand<'tcx>,
         expected: bool,
-        msg: AssertMessage<'tcx>,
+        msg: PanicMessage<_>,
```
Comment (saleemjaffer) [inline src/librustc/mir/mod.rs]: @oli-obk What should the type parameter for `PanicMessage` here?

**Comment by saleemjaffer** [comment]
Closing this, as @RalfJung has a [PR](https://github.com/rust-lang/rust/pull/62927) already

**Comment by RalfJung** [comment]
Oh dang, I did not see this one. :/  I tried searching for PRs but it seems I missed this.

I am so sorry!

**Pull request closed without merging**